test/any/datastore/rate_limited_DAO-test.es6.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('RateLimitedDAO', function() {
+  let rateLimitedDAO;
+  foam.CLASS({
+    package: 'test.RateLimitedDAO',
+    name: 'item',
+    properties: ['id'],
+  });
+  beforeEach(function() {
+    rateLimitedDAO = org.chromium.apis.web.RateLimitedDAO.create({
+      interval: 100,
+      delegate: foam.dao.ArrayDAO.create(),
+    });
+  });
+  it('puts objects at specified interval.', function(done) {
+    rateLimitedDAO.put(test.RateLimitedDAO.item.create({id: 1}));
+    rateLimitedDAO.put(test.RateLimitedDAO.item.create({id: 2}));
+    rateLimitedDAO.put(test.RateLimitedDAO.item.create({id: 3}));
+    setTimeout(() => {
+      rateLimitedDAO.select().then((arraySink) => {
+        expect(arraySink.a.length).toBe(1);
+      });
+    }, 100);
```

Comment:
Let's have some slack time here. Maybe `110` or `150`? Same for others (`setTimeout` some non-negligible time after `put()` should have been issued)